test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -17,6 +16,8 @@ from Standard.Database.Errors import all
 from Standard.Test import Test, Problems
 import Standard.Test.Extensions
 
+import enso_dev.Tests.Data.Round_Spec
```

Comment:
I'm really glad we could get this working 😀 

I think this also opens up doors to more logic sharing where we can import parts of tests from `Tests` in other test suites like `Table_Tests`.